lib/authc/AssertionAuthenticationResult.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var nJwt = require('njwt');
+var Account = require('../resource/Account');
+
+function AssertionAuthenticationResult(dataStore, data) {
```

Comment:
`AuthenticationResult` inherits `InstanceResource` and also have a few different properties and methods. Is this intended to not be the same or why is this different?